tests/conftest.py

```diff
@@ -2,10 +2,33 @@
 import pytest
 
 
+from cachito.web import db as _db
 from cachito.web.wsgi import create_app
+from cachito.web.models import PackageManager
 
 
 @pytest.fixture(scope='session')
-def client():
+def app():
+    """Return Flask application for the pytest session."""
+    return create_app('cachito.web.config.TestingConfig')
+
+
+@pytest.fixture(scope='session')
+def client(app):
     """Return Flask application client for the pytest session."""
-    return create_app('cachito.web.config.TestingConfig').test_client()
+    return app.test_client()
+
+
+@pytest.fixture()
+def db(app):
+    """Yields a DB with required app tables but with no records."""
+    with app.app_context():
+        _db.create_all()
+        # TODO: The default pkg managers are defined in the migration scripts. How
+        # to call those from here?
+        pkg_manager = PackageManager(name='gomod')
+        _db.session.add(pkg_manager)
+        _db.session.commit()
+        yield _db
+        _db.session.remove()
+        _db.drop_all()
```

Comment:
Optional: WaiverDB rolls back the transaction to avoid dropping and recreating the database for every test. This would increase performance of the unit tests:
https://pagure.io/waiverdb/blob/master/f/tests/conftest.py#_46